Model/Api/Products.php

```diff
@@ -40,47 +41,42 @@ public function __construct(
      */
     public function getRichSnippet($productId = null)
     {
+        $return = [
```

Comment:
can you please change the name from return to something a little more meaningful? like rich_snippet_data or something similar?